packages/bindings/binding.gyp

```diff
@@ -62,7 +65,26 @@
             'src/poller.cpp'
           ]
         }
-      ]
-    ]
+      ],
+
+      # Coverage generation options
+      # These coverage options are currently only compatible with non-Windows OSes
+      ['generate_coverage=="yes"',
+        {
+          'cflags+': ['--coverage'],
+          'link_settings': {'libraries+': ['-lgcov']},
+          'xcode_settings': {
+            'GCC_GENERATE_TEST_COVERAGE_FILES': ['YES'],
+            'GCC_INSTRUMENT_PROGRAM_FLOW_ARCS': ['YES'],
+            'OTHER_CFLAGS+': ['-fprofile-arcs -ftest-coverage'],
+            'OTHER_LDFLAGS+': [
+              '-fprofile-arcs -ftest-coverage',
```

Comment:
Theses **all** seem to be necessary instead of the one `--coverage` GCC flag. I've tested without each setting individually.

Way to think different Xcode 😉